main/streams/streams.c

```diff
@@ -839,7 +839,7 @@ PHPAPI int _php_stream_stat(php_stream *stream, php_stream_statbuf *ssb)
 	}
 
 	/* if the stream doesn't directly support stat-ing, return with failure.
-	 * We could try and emulate this by casting to a FD and fstat-ing it,
+	 * We could try and emulate this by casting to an FD and fstat-ing it,
```

Comment:
why?

Reply:
Although "F" is not a vowel, when the letter "F" is read aloud by itself (not as part of a word), it is pronounced as "eff"; i.e. starting with a vowel sound. So many native speakers would naturally use "an" instead of "a".

Reply:
Thanks, every day I learn something new.

Reply:
Pro tip: Sometimes it happens that some native speakers of English read an acronym as a single word, while others read out the individual letters. For example, some read SQL as "sequel" and others as "S Q L".

In such cases, whether "a" or "an" is used may differ; some might say "a SQL query", while others would say "an SQL query". (Can you see which of those is "sequel" and which is "S Q L"?)